src/parser/data/GraphRef.h

```diff
@@ -9,9 +9,18 @@
 #include "parser/Iri.h"
 
 using GraphRef = TripleComponent::Iri;
-struct DEFAULT {};
-struct NAMED {};
-struct ALL {};
+struct DEFAULT {
+  // For testing
+  bool operator==(const DEFAULT&) const = default;
+};
+struct NAMED {
+  // For testing
+  bool operator==(const NAMED&) const = default;
```

Comment:
I am currently not understanding where the nam of the graph comes from, but I am sure that this will become clearer.

Reply:
Some of the update operations operate on all named graphs/the default graph/all graphs. These objects represent these targets. I have added a short comment that (hopefully) explains that.